main.go

```diff
@@ -361,6 +362,16 @@ func init() {
 		}
 	}
 
+	if config.KafkaRest.Address != "" {
+		var err error
+		kafkaRestClient, err = outputs.NewClient("KafkaRest", config.KafkaRest.Address, config.KafkaRest.MutualTLS, config.KafkaRest.CheckCert, config, stats, promStats, statsdClient, dogstatsdClient)
+		if err != nil {
+			config.KafkaRest.Address = ""
+		} else {
+			outputs.EnabledOutputs = append(outputs.EnabledOutputs, "Kafka Rest")
```

Comment:
```suggestion
			outputs.EnabledOutputs = append(outputs.EnabledOutputs, "KafkaRest")
```
outputs are separated by a space when we print the list, this is why we need a string in camel case